### ISSUE TITLE
Extend .travis.yml

### ISSUE THREAD

**Issue by mrakitin** [NSLS-II/scientific-python-cookiecutter#86]
https://github.com/vstadnytskyi/auxiliary/blob/master/.travis.yml can be used as an example of the configuration for Linux/OSX/Windows builds. Tried it in my amateur project https://github.com/mrakitin/extract-pages (with minor modifications), and it worked great! Credits to @vstadnytskyi.

**Comment by danielballan** [comment]
I think this is cool but not in the spirit of a minimal example targeting small- to medium-sized projects managed by scientists whose primary job activity is not programming.

**Comment by vstadnytskyi** [comment]
some codes I write are related to data acquisition and server-client interactions (these days I mostly use caproto and pyepics for that). I have had issues with some codes working during development and don't work after deployment. Which was the pain to debug since I had to sit in front(or VNC) to a different machine. It came down to some codes working differently on different platforms. I develop on macOS and deploy it to Windows or Linux. After putting together a multiplatform configuration for Travis-CI, I was able to extensively test codes before deployment.

I thought this extension might be useful for some users. Now, we have an issue, and if someone will need they will be able to find my config file :)
